app/components/post/post.js

```diff
@@ -307,6 +308,11 @@ class Post extends PureComponent {
             showFullDate,
             theme
         } = this.props;
+
+        if (!post) {
```

Comment:
When are we rendering a post component without a Post?

Reply:
haven't track this one but there are some times that the post is null and that was causing a crash, so better safe than sorry I guess